packages/component-library/src/MapOverlay/MapOverlay.js

```diff
@@ -0,0 +1,99 @@
+import React, { PropTypes, Component } from 'react';
+import MapGL from 'react-map-gl';
+import { css } from 'emotion';
+import './mapbox-gl.css';
+import DeckGLOverlay from './../../deckgl-overlay.js';
+
+const mapWrapper = css`
+  margin: auto;
+  max-width: 900px;
+`;
+
+const colorScale = r => [r * 255, 140, 200 * (1 - r)];
+
+// Source data GeoJSON (currently only Vancouver BC)
+const DATA_URL =
+  'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/geojson/vancouver-blocks.json'; // eslint-disable-line
+
+export default class BaseMap extends Component {
+  constructor(props) {
+    super(props);
```

Comment:
Fun trivia fact: super(props) isn't needed unless you're trying to call `this.props` within the constructor or you're calling a separate class method within the constructor that also tries to reference `this.props`